rdf4j/src/main/java/eu/neverblink/jelly/convert/rdf4j/rio/JellyParser.java

```diff
@@ -113,11 +113,7 @@ public void handleTriple(Value subject, Value predicate, Value object) {
             if (delimitingResponse.isDelimited()) {
                 // Delimited Jelly file
                 // In this case, we can read multiple frames
-                readStream(
-                    delimitingResponse.newInput(),
-                    inputStream -> ProtoMessage.parseDelimitedFrom(inputStream, getReusableFrame),
-                    frame -> buffer.clear()
-                );
+                readStream(delimitingResponse.newInput(), getReusableFrame, frame -> buffer.clear());
```

Comment:
so parseDelimitedFrom instantiates a new CodedIS for each call?

Reply:
Yes. That's how it was in every protobuf library I've seen in Java/Scala. They simply have no native support for this whole "delimited stream" thing, as this is not a part of the protobuf spec.